src/components/WwwFrame/WwwPage.vue

```diff
@@ -29,6 +29,10 @@ export default {
 	display: flex;
 	flex-flow: column nowrap;
 
+	& > * {
```

Comment:
@emuvente maybe add a comment here about what this does...? Just for future reference

Reply:
I think it might be better as `header, main, footer` anyway